docs/source/fixtures.rst

```diff
@@ -47,3 +47,35 @@ Fixtures receive a provider and an array of wallets as an argument. By default,
     // fixture implementation
   });
 
+Experimental version with using dockerized vyper for compilation.
```

Comment:
Wrong place, should be in configuration file, should be introduced in [compilerType](https://github.com/EthWorks/Waffle/blob/0f6d5d6b5b9097fa29ae59a7710a177b061e3f73/docs/source/configuration.rst#compilertype) and explained in detail in separate section below.